<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1433</width>
    <height>925</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>98</green>
        <blue>108</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>79</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>63</green>
        <blue>72</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>98</green>
        <blue>108</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>79</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>63</green>
        <blue>72</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>43</red>
        <green>98</green>
        <blue>108</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>79</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>63</green>
        <blue>72</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>25</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Avenir Next</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>BMW Dealer Manager</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Highlight">
            <brush brushstyle="SolidPattern">
             <color alpha="171">
              <red>28</red>
              <green>114</green>
              <blue>202</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Highlight">
            <brush brushstyle="SolidPattern">
             <color alpha="171">
              <red>28</red>
              <green>114</green>
              <blue>202</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Highlight">
            <brush brushstyle="SolidPattern">
             <color alpha="170">
              <red>31</red>
              <green>117</green>
              <blue>204</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Avenir Next</family>
         </font>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Salon</string>
         </attribute>
         <widget class="QPushButton" name="Savetofile">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>720</y>
            <width>1411</width>
            <height>51</height>
           </rect>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>104</red>
                <green>96</green>
                <blue>40</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>104</red>
                <green>96</green>
                <blue>40</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>104</red>
                <green>96</green>
                <blue>40</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>25</red>
                <green>25</green>
                <blue>25</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="NoRole">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>33</red>
                <green>255</green>
                <blue>6</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>104</red>
                <green>96</green>
                <blue>40</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>104</red>
                <green>96</green>
                <blue>40</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>104</red>
                <green>96</green>
                <blue>40</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>25</red>
                <green>25</green>
                <blue>25</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="NoRole">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>33</red>
                <green>255</green>
                <blue>6</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>104</red>
                <green>96</green>
                <blue>40</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>104</red>
                <green>96</green>
                <blue>40</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>104</red>
                <green>96</green>
                <blue>40</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="170">
                <red>31</red>
                <green>117</green>
                <blue>204</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="NoRole">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>33</red>
                <green>255</green>
                <blue>6</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Zapisz obecny stan salonu</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>1361</width>
            <height>551</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="pics.qrc">:/Resources/image2.jpg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="Wyjscie">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>770</y>
            <width>1411</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Wyjście</string>
          </property>
         </widget>
         <widget class="QLabel" name="stanKontsa">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>590</y>
            <width>301</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="SalonInfo">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>670</y>
            <width>1411</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Wyświetl informacje o salonie</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Samochody</string>
         </attribute>
         <widget class="QPushButton" name="Sprzedaj_Auto">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>770</y>
            <width>1391</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sprzedaj samochód</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Posiadane_samochodu">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>670</y>
            <width>1391</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Posiadane samochody</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Dodaj_samochod">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>720</y>
            <width>1391</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Kup Samochód</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>1201</width>
            <height>651</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="pics.qrc">:/Resources/image3.jpg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="stanKontsa_2">
          <property name="geometry">
           <rect>
            <x>980</x>
            <y>600</y>
            <width>301</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>18</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_7">
         <attribute name="title">
          <string>Motocykle</string>
         </attribute>
         <widget class="QPushButton" name="Posiadane_motocykle">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>680</y>
            <width>1391</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Posiadane motocykle</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>720</y>
            <width>1391</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Kup motocykl</string>
          </property>
         </widget>
         <widget class="QPushButton" name="SprzedajMotocykl">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>760</y>
            <width>1391</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir Next</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sprzedaj motocykl</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>10</y>
            <width>1141</width>
            <height>661</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="pics.qrc">:/Resources/image4.jpg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="stanKontsa_3">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>610</y>
            <width>301</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>18</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="Personnel_tab">
         <attribute name="title">
          <string>Personel</string>
         </attribute>
         <widget class="QPushButton" name="Zatrudnieni_pracownicy">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>670</y>
            <width>1391</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Zatrudnieni pracownicy</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Zatrudnij_nowego">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>720</y>
            <width>1391</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Zatrudnij nowego pracownika</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Zwolnij_pracownika">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>771</y>
            <width>1391</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Avenir</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Zwolnij pracownika</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>114</x>
            <y>10</y>
            <width>1201</width>
            <height>533</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="pics.qrc">:/Resources/image5.jpg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1433</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
